### PR TITLE
feat: rename helix to franklin

### PR DESCRIPTION
**Pull request by tripodsan**
- renames helix to franklin in most of the output
- replace helix pages logo
- replace helix importer logo
- rename `--pages-url` to `--url` (keep backwards compat)

image

image

### CODE REVIEW

File: src/server/Indexer.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ export default class Indexer {
       log.error(`Error in helix-query.yaml: ${e.message}`);
     }
     if (this._printIndex && this._index.indices.length === 0) {
-      log.warn(chalk`Helix CLI started with {gray --print-index} but no valid {gray helix-query.yaml} found.`);
+      log.warn(chalk`Franklin CLI started with {gray --print-index} but no valid {gray franklin-query.yaml} found.`);
```
Comment (kptdobe) [inline src/server/Indexer.js:78]: `helix-query.yaml` really becomes `franklin-query.yaml` ??

**Review by kptdobe — CHANGES_REQUESTED** [review]

File: src/server/Indexer.js (hunk 1 above)
Comment (tripodsan) [reply src/server/Indexer.js:78]: oops... too much search/replace

**Review by kptdobe — APPROVED** [review]

File: src/server/HelixImportServer.js [hunk 2]
```diff
@@ -235,8 +235,8 @@ export default class HelixServer extends EventEmitter {
       // codecov:ignore:start
       /* c8 ignore start */
       } catch (err) {
-        log.error(`Failed to proxy helix request ${ctx.path}: ${err.message}`);
-        res.status(502).send(`Failed to proxy helix request: ${err.message}`);
+        log.error(`Failed to proxy franklin request ${ctx.path}: ${err.message}`);
```
Comment (trieloff) [inline src/server/HelixImportServer.js:238]:
```suggestion
        log.error(`Failed to proxy Franklin request ${ctx.path}: ${err.message}`);
```

File: src/server/HelixImportServer.js [hunk 3]
```diff
@@ -235,8 +235,8 @@ export default class HelixServer extends EventEmitter {
...
+        res.status(502).send(`Failed to proxy franklin request: ${err.message}`);
```
Comment (trieloff) [inline src/server/HelixImportServer.js:239]:
```suggestion
        res.status(502).send(`Failed to proxy Franklin request: ${err.message}`);
```

File: src/server/HelixImportServer.js [hunk 4]
```diff
@@ -280,7 +280,7 @@ export default class HelixServer extends EventEmitter {
         throw new Error(`Port ${this._port} already in use by another process.`);
       }
     }
-    log.info(`Starting helix import server v${packageJson.version}`);
+    log.info(`Starting franklin import server v${packageJson.version}`);
```
Comment (trieloff) [inline src/server/HelixImportServer.js:283]:
```suggestion
    log.info(`Starting Franklin import server v${packageJson.version}`);
```

File: src/server/HelixProject.js [hunk 5]
```diff
@@ -170,7 +170,7 @@ export default class HelixProject {
   }
 
   async start() {
-    this.log.debug('Launching helix simulation server for development...');
+    this.log.debug('Launching franklin simulation server for development...');
```
Comment (trieloff) [inline src/server/HelixProject.js:173]:
```suggestion
    this.log.debug('Launching Franklin simulation server for development...');
```

File: src/server/HelixProject.js [hunk 6]
```diff
@@ -179,7 +179,7 @@ export default class HelixProject {
   }
 
   async stop() {
-    this.log.debug('Stopping helix simulation server..');
+    this.log.debug('Stopping franklin simulation server..');
```
Comment (trieloff) [inline src/server/HelixProject.js:182]:
```suggestion
    this.log.debug('Stopping Franklin simulation server..');
```

File: src/server/HelixServer.js [hunk 7]
```diff
@@ -99,8 +99,8 @@ export default class HelixServer extends EventEmitter {
         cacheDirectory: this._project.cacheDirectory,
       });
     } catch (err) {
-      log.error(`Failed to proxy helix request ${ctx.path}: ${err.message}`);
-      res.status(502).send(`Failed to proxy helix request: ${err.message}`);
+      log.error(`Failed to proxy franklin request ${ctx.path}: ${err.message}`);
```
Comment (trieloff) [inline src/server/HelixServer.js:102]:
```suggestion
      log.error(`Failed to proxy Franklin request ${ctx.path}: ${err.message}`);
```

File: src/server/HelixServer.js [hunk 8]
```diff
@@ -99,8 +99,8 @@ export default class HelixServer extends EventEmitter {
...
+      res.status(502).send(`Failed to proxy franklin request: ${err.message}`);
```
Comment (trieloff) [inline src/server/HelixServer.js:103]:
```suggestion
      res.status(502).send(`Failed to proxy Franklin request: ${err.message}`);
```

**Review by trieloff — COMMENTED** [review]
Consistent upper-case Franklin

**Review by trieloff — APPROVED** [review]

**Review by rofe — APPROVED** [review]

**Pull request merged**